### ISSUE TITLE
✨ Feature Suggestion | Add software/covid-19 page

### ISSUE THREAD

**Issue by ph00lt0** [privacytools/privacytools.io#2353]
## Description
In accordance with comments on issue #2301 @ lrq3000 and me would suggest to add a collection of covid-19 apps. Covid-19 apps are often not privacy friendly but required to join venues. On F-droid there are already some alternative apps which could replace the the more evil once. This will help people not to get excluded from society and being able to protect their privacy.

It might also be nice to add apps that could store QR codes like Catima.

**Comment by lrq3000** [comment]
Yes I support this suggestion. I can make a PR (it will be the first time I design a new section but it will be a good training) if @editorial sees value in this proposition.

I would also suggest that this section adds mentions of how to uninstall automatically installed covid apps such as [what Google did with Android phones in Massachusetts](https://www.reddit.com/r/privacy/comments/o41la4/massachusetts_covid19_tracking_app_autoinstalls/).

**Comment by freddy-m** [comment]
@privacytools/editorial I like this idea - what are your thoughts?

**Comment by Super-Macintosh** [comment]
Good idea
